calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
+        print "Couldn't connect to gobgp."
+        return
+
+    for neighbor in neighbors:
+        # calico-bgp-daemon currently only supports node-to-node mesh type
```

Comment:
I haven't implemented other types in `calico-bgp-daemon` yet.
I can implement other types after this PR is merged.

Added a warning https://github.com/projectcalico/calico-containers/pull/1126/commits/c7127fc5338b4460f1d5a14e617e5158b2a3c08f anyway.